src/test/java/com/comcast/zucchini/FastrunZucchiniCukesTest.java

```diff
@@ -42,12 +43,12 @@
 class FastrunZucchiniCukesTest extends AbstractZucchiniTest {
     private static final Logger LOGGER = LoggerFactory.getLogger(FastrunZucchiniCukesTest.class);
 
+    private String[] inputs = new String[]{"asparagus", "carrots", "potato"};
```

Comment:
It does not seem that you are using these veggies anywhere anymore. So, you might as well get rid of them here and get rid of them here and on lines 53-54